packages/lambda-at-edge/src/default-handler.ts

```diff
@@ -67,6 +67,20 @@ const router = (
 };
 
 const normaliseUri = (uri: string): string => (uri === "/" ? "/index" : uri);
+const normaliseS3OriginDomain = (s3Origin: CloudFrontS3Origin): string => {
+  if (s3Origin.region === "us-east-1" || !s3Origin.region) {
+    return s3Origin.domainName;
+  }
+
+  if (!s3Origin.domainName.includes(s3Origin.region)) {
+    return s3Origin.domainName.replace(
+      "s3.amazonaws.com",
+      `s3-${s3Origin.region}.amazonaws.com`
```

Comment:
We shouldn't use this pattern according to: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html#access-bucket-intro
`s3.${s3Origin.region}.amazonaws.com`

Reply:
good catch, updated!